docs/source/models/intervention_models/intrapartum/azithromycin_intervention.rst

```diff
@@ -0,0 +1,265 @@
+related to the facility choice during birth, i.e. home births have much lower access than in-facility births, and births in BEmONC facilities have lower access than CEmONC 
+facilities.
+
+To make this work naturally with the ``RiskEffect`` component, it is best to think of the risk as "lack of access to azithromycin".  With this framing, the ``RiskEffect`` 
+component requires data on (1) the relative risk of sepsis incidence for people with lack of access to azithromycin, and (2) the population attributable fraction (PAF) of sepsis 
+due to lack of access to azithromycin.  We will use the decision tree below to find the probability of sepsis incidence with and without access to azithromycin that are logically 
+consistent with the baseline delivery facility rates and baseline azithromycin coverage.
+
+In Vivarium, this risk effect will modify the sepsis incidence pipeline, resulting in 
+
+.. math::
+
+   \text{IR}_i^\text{sepsis} = \text{IR}^\text{sepsis}_{\text{BW}_i, \text{GA}_i} \cdot (1 - \text{PAF}_\text{no azithromycin}) \cdot \text{RR}_i^\text{no azithromycin}
+
+where :math:`\text{RR}_i^\text{no azithromycin}` is simulant *i*'s individual relative risk for "no azithromycin", meaning :math:`\text{RR}_i^\text{no azithromycin} = \text{RR}_\text{no azithromycin}` 
+if simulant *i* accesses a facility without azithromycin, and :math:`\text{RR}_i^\text{no azithromycin} = 1` if simulant *i* accesses a facility *with* azithromycin.
+
+If there are other interventions also affecting the IR of sepsis, the pipeline will combine these effects, and we can write out the math for this risk explicitly as 
+
+.. math::
+
+   \text{IR}^\text{sepsis}_{i, \text{updated}} = \text{IR}^\text{sepsis}_{i, \text{original}} \cdot (1 - \text{PAF}_\text{no azithromycin}) \cdot \text{RR}_i^\text{no azithromycin}
+
+This reduces to the previous formula if there are no other interventions, and we would have 
+
+.. math::
+
+   \text{IR}^\text{sepsis}_{i, \text{original}} = \text{IR}^\text{sepsis}_{\text{BW}_i, \text{GA}_i}
+
+The relative risk value we will use is pulled from `this 2024 systematic review/meta-analysis <https://bmcpregnancychildbirth.biomedcentral.com/articles/10.1186/s12884-024-06390-6#:~:text=Primary%20outcomes,-Among%20the%20six&text=The%20incidence%20of%20maternal%20sepsis%20was%20significantly%20lower%20in%20the,was%20analysed%20in%20three%20studies.>`_ 
+that investigated the effect of azithromycin during labor.
+
+.. list-table:: Risk Effect Parameters for Lack-of-Access-to-Azithromycin
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Parameter
+    - Mean
+    - Distribution
+    - Notes
+  * - Relative Risk
+    - 1.54
+    - :math:`\text{Normal}(1.54,0.08^2)`
+    - Based on placeholder relative risk of 0.65 (95% CI 0.55-0.77) on sepsis incidence for pregnant people with access to azithromycin
```

Comment:
Let's make sure the reference get's directly recorded as a citation here so it doesn't get lost.